packages/components/src/components/interface/ListTable/ListTable.tsx

```diff
@@ -71,9 +71,10 @@ ${({ fixedWidth, totalWidth }) =>
 
 const TableHeaderText = styled.div<{
   isSortable?: boolean
+  isSorted?: boolean
 }>`
-  ${({ isSortable, theme }) =>
-    isSortable ? theme.fonts.bodyBoldStyle : theme.fonts.bodyStyle}
+  ${({ isSortable, isSorted, theme }) =>
+    isSortable && isSorted ? theme.fonts.bodyBoldStyle : theme.fonts.bodyStyle}
```

Comment:
`isSortable` is redundant here actually as if `isSorted` is true, `isSortable` will also be true.

Reply:
Removed It !